### ISSUE TITLE
Unable to reproduce the experiment results on the PoseBusters set

### ISSUE THREAD

**Issue by myhDaniel** [tencent-ailab/Interformer#6]
Hello, thank you for your excellent work! I recently attempted to reproduce your results on the PoseBusters dataset but was unable to achieve the reported performance. I directly used the data and model checkpoints you provided and ran the bash script `scripts/infer/docking_posebuster.sh` the used the PoseBusters tool to check the generated molecule pose.

However, the results I obtained were different from those reported in your paper. Specifically, the percentage of top-1 RMSDs below 2 Å was 76.64%, and RMSD < 2 Å & PB-valid was 76.17%, compared to the reported 84.09% and 76.29%, respectively.  

Could you please inform me about any potential mistakes I might have made during this process? Thank you!

**Comment by lkfo415579** [comment]
@myhDaniel Did you use all the docking pose to evaluate the RMSD (n=428)?  The performance of  Posebuster version 1 is slightly lower than the version 2. (SOTA papers use version2 nowadays) We have uploaded the related split data. For more details on how to evaluate the RMSD, please refer to script [eval_docking_ac.py](https://github.com/tencent-ailab/Interformer/blob/master/eda/docking/eval_docking_ac.py)

**Comment by myhDaniel** [comment]
Thanks for your guidance! I did use all the docking poses before. This time, I ran your script [eval_docking_ac.py](https://github.com/tencent-ailab/Interformer/blob/master/eda/docking/eval_docking_ac.py) and obtained the following output:

[input_csv] [poserank2exclude]
Excluding pose_rank = 20 (crystal structure)
Full
Using Energy for ranking.
N:428
PoseRank-top1 328 / 428 -> 76.6355140186916
PoseRank-top1_median 0.9899295
N:428
PoseRank-top5 400 / 428 -> 93.45794392523365
PoseRank-top5_median 0.8047445
N:428
PoseRank-top10 406 / 428 -> 94.85981308411215
PoseRank-top10_median 0.7704665
\**************************************************
\**************************************************
Posebuster v2
Using Energy for ranking.
N:308
PoseRank-top1 248 / 308 -> 80.51948051948052
PoseRank-top1_median 0.9230485
N:308
PoseRank-top5 300 / 308 -> 97.40259740259741
PoseRank-top5_median 0.7431105
N:308
PoseRank-top10 300 / 308 -> 97.40259740259741
PoseRank-top10_median 0.7149665000000001
\**************************************************
\**************************************************

I then used PoseBusters tool to check the generated molecule poses again(only check the 308 cases). The new results I got are as followed (with your reported results in parentheses):
- RMSD<2：80.52%（ 84.09%）
- RMSD<2 & PB-valid: 79.55% (76.29%)

What might be the reasons that cause the difference?

**Comment by lkfo415579** [comment]
The differences may be due to the use of different seeds in the docking program. Try rerunning it to verify the results. However, it's uncommon to see a significant difference in performance caused by different seeds.

**Comment by myhDaniel** [comment]
Yes, as you mentioned, it’s probably due to the seeds. I reran the script, and the new results I obtained are as follows (with your reported results in parentheses):
- RMSD<2：83.77%（ 84.09%）
- RMSD<2 & PB-valid: 72.73% (76.29%)

The results seem more plausible this time. However, how can the difference be explained when considering PB-valid?
